addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -119,6 +121,8 @@ public class SqlInjectionScanRule extends AbstractAppParamPlugin {
      */
     private static final String[] SQL_CHECK_ERR = {"'", "\"", ";", "'(", ")", "(", "NULL", "'\""};
 
+    private String locationHeaderNormal;
```

Comment:
Does this need to be a class member? It's currently redeclared at 946.

Reply:
it was used in `expressionBasedAttack` method. let me refactor it as argument